src/NzbDrone.Core.Test/ParserTests/ParserFixture.cs

```diff
@@ -238,29 +240,17 @@ public void should_parse_tmdb_id(string postTitle, int tmdbId)
 
         [TestCase("The.Italian.Movie.2025.720p.BluRay.X264-AMIABLE")]
         [TestCase("The.French.Movie.2013.720p.BluRay.x264 - ROUGH[PublicHD]")]
+        [TestCase("The.German.Doctor.2013.LIMITED.DVDRip.x264-RedBlade", Description = "When German is not followed by a year or a SCENE word it is not matched")]
+        [TestCase("The.Good.German.2006.720p.HDTV.x264-TVP", Description = "The Good German is hardcoded not to match")]
+        [TestCase("German.Lancers.2019.720p.BluRay.x264-UNiVERSUM", Description = "German at the beginning is never matched")]
+        [TestCase("The.German.2019.720p.BluRay.x264-UNiVERSUM", Description = "The German is hardcoded not to match")]
         public void should_not_parse_wrong_language_in_title(string postTitle)
         {
             var parsed = Parser.Parser.ParseMovieTitle(postTitle, true);
             parsed.Languages.Count.Should().Be(1);
             parsed.Languages.First().Should().Be(Language.Unknown);
         }
 
-        [TestCase("The.Movie.Name.2016.German.DTS.DL.720p.BluRay.x264-MULTiPLEX")]
-        public void should_not_parse_multi_language_in_releasegroup(string postTitle)
-        {
-            var parsed = Parser.Parser.ParseMovieTitle(postTitle, true);
-            parsed.Languages.Count.Should().Be(1);
-            parsed.Languages.First().Should().Be(Language.German);
-        }
-
-        [TestCase("The.Movie.Name.2016.German.Multi.DTS.DL.720p.BluRay.x264-MULTiPLEX")]
```

Comment:
No, because the logic of this test has been removed in https://github.com/Radarr/Radarr/commit/ad8629fac981217f5a4a5068da968c29d9ee634c and later the test was changed in https://github.com/Radarr/Radarr/commit/7ec0fd1cea5e6203a6f7acbd3de5354c24193dec so it passed again, but it was not the intended result of the initial test anymore. The code that it was initially tested against is missing.
The new Multi language logic has then been implemented in https://github.com/Radarr/Radarr/commit/efd4abfa3eaab0b476ff8d69a7ef4fc98ae1fd7d again, but there is currently no test for it.

Reply:
I don't think so. Searching for `German DL ML` on [xrel.to](https://www.xrel.to/releases.html?search_results=cOsRCcO9DG167582203&query=german+ml+dl&maxed_out=0) doesn't show any result, only an episode of an Anime with `Ml` in the episode title. It wouldn't even make much sense, because `ML` (German, Original, English) includes `DL` (German, Original).
I used `DL` first because it is widely used, `GermanDualLanguageRegex` matches much more often than `GermanMultiLanguageRegex`. But if you prefer, I can switch the statements, so that `ML` is preferred, if both are being found in future releases (if any). It would be more correct in that case.